Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -15,12 +15,52 @@ public Inventory()
 
         public Item[] GetItems(string name)
         {
-            return new Item[0];
+            var numberOfMatchName = GetNumberOfMatchName(name);
+            if (numberOfMatchName == 0)
+            {
+                return new Item[0];
+            }
+
+            var tempItems = new Item[numberOfMatchName];
+            var i = 0;
+
+            foreach (var item in _items)
+            {
+                if (item.GetName() == name)
+                {
+                    tempItems[i++] = item;
+                }
+            }
+
+            return tempItems;
+        }
+
+        private int GetNumberOfMatchName(string name)
```

Comment:
Number is vague. Call "GetMatchingNamesCount"

Reply:
done